.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+- id: nb-clean
+  name: nb-clean
+  entry: 'nb-clean clean'
+  require_serial: true
```

Comment:
For my understanding, why do we need `require_serial`?

Reply:
I initially thought this referred to running multiple hooks in parallel, but realized it's just concurrency for the hook itself. So probably fine to delete it, could be a small speedup for some people!